autoware_adapi_v1_msgs/perception/msg/Shape.msg

```diff
@@ -0,0 +1,8 @@
+uint8 UNKNOWN=0
+uint8 BOUNDING_BOX=1
+uint8 CYLINDER=2
+uint8 POLYGON=3
```

Comment:
```suggestion
uint8 PRISM=3
```

Reply:
from autoware_auto_perception_msgs, the shape is using `POLYGON`, I think to prevent confuse, we should keep the same?

Reply:
This actually means PRISM (it uses `dimension.z` as height). I think it's better to change `autoware_auto_perception_msgs` to avoid confusion.

Reply:
I see, do you mean we also need to change the autoware_auto_msgs?

Reply:
Not needed right now as it affects some nodes.

Reply:
okay, then I just change in api_msgs

Reply:
changed at 
f6bfa5e